src/omv/modules/py_helper.c

```diff
@@ -181,7 +181,7 @@ void py_helper_arg_to_float_array(const mp_obj_t arg, float *array, size_t size)
     }
 }
 
-image_t *py_helper_keyword_to_image(uint n_args, const mp_obj_t *args, uint arg_index,
+image_t *py_helper_keyword_to_image(size_t n_args, const mp_obj_t *args, uint arg_index,
```

Comment:
@kidswong999 In this file `py_helper.c` and `py_helper.h` we should also change `arg_index` from `uint` to `size_t`
```suggestion
image_t *py_helper_keyword_to_image(size_t n_args, const mp_obj_t *args, size_t arg_index,
```